src/CMakeLists.txt

```diff
@@ -296,7 +296,8 @@ else()
 		if(NOT HAIKU)
 			find_package(X11 REQUIRED)
 		endif(NOT HAIKU)
-		set(OpenGL_GL_PREFERENCE "GLVND")
+		# Use LEGACY because GLVND breaks several NVIDIA drivers
```

Comment:
this comments actually makes it worse because it's inaccurate
try `# Use LEGACY because GLVND is broken on some nvidia setups, see #7955 and #7665`